react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1027 @@
+                <td>Harder (.apk or .ipa)</td>
+                <td>Easy (QR-code)</td>
+              </tr>
+            </tbody>
+          </table>
+          <aside class="notes">
+            Well, as you can see, the selection of the CLI is dependent on the requirements and feature set that is to be implemented.
+Given a comparison of the cons and props of both frameworks, I would suggest screening through the requirements and choosing which one suits your application most.
+          </aside>
+        </section>
+
+        <section class="picture-slide">
+          <h4 class="slide-name">Alternatives</h4>
+          <img src="assets/images/alternatives.png" alt="simple" />
+          <aside class="notes">
+            React Native is not the only one technology, that allow you to build mobile apps for both iOS and Android. You can also do it with Flutter and Ionic. 
+          </aside>
+        </section>
+
+        <section class="bullets-and-picture-slide">
+          <h4 class="slide-name">Flutter</h4>
+          <div>
+            <img src="assets/images/flutter_big.png" alt="simple" />
+            <div class="pros-cons-section">
+              <p class="big">
+                Pros:
+              </p>
+              <ul>
+                <li>UI building components, Widgets</li>
+                <li>Dart support</li>
+                <li>Perfomance</li>
+              </ul>
+              <p class="big">
+                Cons:
+              </p>
+              <ul>
+                <li>Scarce ecosystem</li>
+                <li>Design features for iOS may not be up to date</li>
+              </ul>
+            </div>
+          </div>
+          <aside class="notes">
+            Powered by Google, Flutter designs native-type applications on Android and iOS. Flutter was launched in 2017 and consists of widgets as its building blocks. It uses Dart, which is developed by Google as well. Dart is based on C/C++.<br><strong>Flutter advantages.</strong> Flutter ships with a <strong>comprehensive suite</strong> of built-in widgets. With these widgets, you can quickly create nice-looking UIs without doing too much manual styling. UI building components, Widgets, are rendered only when necessary that resembles a declarative approach applied in React. <strong>Flutter</strong> use Dart to develop high-quality applications for Android, iOS and the web. Dart supports both AOT(Ahead-of-time) and JIT(Just-in-time) compilation that improves app startup times and overall performance.<br><strong>Disadvantages of Flutter.</strong> \tScarce ecosystem. Despite impressive support by Google and numerous helpful libraries for Flutter, the framework still lacks a rich set of ready-to-use solutions to cover most functionalities. Also, Flutter allows you to enjoy building Android apps fast and efficient. However, design features of some widgets for iOS may not be up to date, which is an issue from a time-efficiency perspective.<br>
```

Comment:
it's better to say "Widgets are rendered only when necessary which resembles a declarative approach applied in React."
"<strong>Flutter</strong> uses Dart to develop high-quality applications for Android, iOS and the web."
\tScarce ecosystem. (remove \t)
Why this sentence is among the disadvantages? "Also, Flutter allows you to enjoy building Android apps fast and efficient."

Reply:
Done